app/services/payout_report_publisher_includer.rb

```diff
@@ -29,11 +29,9 @@ def perform
 
     @publisher.channels.verified.each do |channel|
       probi = wallet.channel_balances[channel.details.channel_identifier].probi # probi = balance - fee
```

Comment:
IMO the readability could be better where the first part is clearly the code looking through the publisher's balance and then looking at all of the publisher's channel balances.

Otherwise, it's good :+1:

Reply:
I agree.  This is because the owner/referral balance in the `channel_balances` hash in the wallet.

We rename the `channel_balances` array in wallet to `account_balances` in https://github.com/brave-intl/publishers/pull/1438.